lib/order-by-first-call.js

```diff
@@ -0,0 +1,16 @@
+"use strict";
+
+var sort = require("./prototypes/array").sort;
+var slice = require("./prototypes/array").slice;
+
+module.exports = function orderByFirstCall(spies) {
+    return sort(slice(spies), function(a, b) {
```

Comment:
You could extract a `comparator` function here to show that `spies` isn't used in the function (and also to not create the function more than once).

Reply:
I've pushed another commit that does this. Once we're happy with the PR, I'll rebase and merge it